Sources/UIView+animate.swift

```diff
@@ -73,9 +73,7 @@ extension UIView {
 
     static func completePendingAnimations() {
         layersWithAnimations.forEach {
-            // run pending animations which are supposed to finish within the next 60 minutes
-            let oneHourInMs: Double = 60 * 60 * 1000
```

Comment:
it turned out `60 * 60 * 1000` is still bigger than a long which is a Int32 on 32bit arch. @ephemer please correct me if this is wrong

Reply:
we changed `NSDate.distantFuture.timeIntervalSince1970` to `60 * 60 * 1000` which is still too big

Reply:
Int32.MAX == 2^31 - 1 == 2_147_483_647, which is quite a bit bigger than 3.6 million (60 * 60 * 1000). So if we're counting on that, I don't think the change in this PR will fix the issue :(

Reply:
we transform  `60 * 60 * 1000` in ms to ns by multiplying it again with 1000. this is causing the crash

60 * 60 * 1000 * 1000 which is 3_600_000_000

Reply:
ahh, ok, but that will be fixed by using whole seconds in the first instance. So the fix will still work, but the github comment is a bit confusing

Reply:
fair point @ephemer . I missed the conversion from ms to ns in the first comment.

Reply:
@ephemer do you think we should rename the timer argument `startingAt` to `startingAtFromNow`? As far as I understand `gettimeofday` sets the timer to now and then we may increment it by passing an argument. This why I changed the call to `$0.animate(at: Timer(startingAt: NSDate.distantFuture.timeIntervalSinceNow))` (instead of timeIntervalSince1970)

Reply:
I don't really have an opinion about the naming. Can you give an idea what the Timer is used for there? The API looks unfamiliar to me

Reply:
but we can discuss this in another iteration. I'm going to merge this pr now

Reply:
Ok fair enough. To be honest `animate(at:` sounds like it should be receiving a timestamp since 1970 (or on Apple platforms, since 2001) and it is pretty confusing to me that we have a `Timer` in there at all, which to me is something that should be measuring a time _interval_ (duration) rather than defining a specific point (instant) in time.

All of this will go away with the work by Swift itself on defining these concepts in the language – I would suggest we just leave it for now and use the official APIs once they become available. https://github.com/apple/swift-evolution/blob/main/proposals/0329-clock-instant-duration.md